app.yaml

```diff
@@ -6,14 +6,15 @@ includes:
 
 handlers:
   - url: /api/.*
```

Comment:
For everyone, this means all back-end routes need to start with api (just in case you run into weird bugs in the future, it's good to know that)

Reply:
Correct.

@Disquietude @airensurzyn as FYI